next-docs/pages/contribute.mdx

```diff
@@ -0,0 +1,37 @@
+## Contribute
+
+We'd love your support in improving MUD! This monorepo includes all of MUD's source code, and pull requests are always welcome. To discuss new features or changes [join our Discord](https://lattice.xyz/discord).
+
+### Local development setup
+
+The following steps are only necessary if you want to contribute to MUD. To use MUD in your project, [set up a new project with the MUD cli](/quick-start).
+
+1. Ensure you have these applications installed:
+
+   | Software                                                                                    | Tested with version |
+   | ------------------------------------------------------------------------------------------- | ------------------- |
+   | [`curl`](https://curl.se/)                                                                  | 7.74.0              |
+   | [`node`](https://nodejs.org/en/download/package-manager)                                    | 18.17.0             |
+   | [`pnpm`](https://pnpm.io/)                                                                  | 8.6.11              |
+   | [`go` (at least version 1.20)](https://go.dev/dl/)                                          | 1.20.7              |
+   | [`make`](https://www.gnu.org/software/make/)                                                | 4.3                 |
+   | [`protoc`](https://grpc.io/docs/protoc-installation/#install-pre-compiled-binaries-any-os/) | 3.15.8              |
```

Comment:
```suggestion
```

these shouldn't be needed anymore since the `services` package is deprecated and not included in the build process anymore